forge/test/mlir/test_ops.py

```diff
@@ -14,6 +14,74 @@
 from forge.tensor import to_forge_tensors, to_pt_tensors
 
 
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        (1, 96, 54, 54),
+        (1, 64, 55, 54),
+        (1, 1, 32, 32),
+        (1, 128, 26, 26),
+        (1, 256, 26, 26),
+    ],
+)
+@pytest.mark.xfail(
+    reason="error: 'ttnn.max_pool2d' op ttnn.max_pool2d currently only supports an input type of bfloat16. Recieved 'f32'."
+)
+@pytest.mark.push
+def test_maxpool2d_with_ceil_mode(input_shape):
```

Comment:
Can we have a single maxpool 2d test and keep cail as pytest parameters?

Reply:
are you referring to combine this [maxpool2d](https://github.com/tenstorrent/tt-forge-fe/blob/13d2afcc7a6f28fef74623b29803845b87a028f9/forge/test/mlir/test_ops.py#L1616) with my test?

Reply:
Ideally, it should be single test_max_pool_2d, and rest should be parameters if possible :))

Reply:
Modified !